app/src/components/market/sections/market_list/market_home.tsx

```diff
@@ -318,9 +331,11 @@ export const MarketHome: React.FC<Props> = (props: Props) => {
     }
   })
 
-  const filterItems: Array<DropdownItemProps> = filters.map(item => {
+  const filterItems: Array<DropdownItemProps> = filters.map((item, index) => {
+    const count = index === 0 ? counts.open : index === 3 ? counts.closed : 0
```

Comment:
Looks like this only displays the number of open and ended categories, and not for pending or finalized

Reply:
@KadenZipfel yeah it is actually not easy to get those numbers from the subgraph I believe!